public/app/features/alerting/unified/RedirectToRuleViewer.tsx

```diff
@@ -22,7 +24,7 @@ function useRuleFindParams() {
   // Relevant issue: https://github.com/remix-run/history/issues/505#issuecomment-453175833
   // It was probably fixed in React-Router v6
   const location = useLocation();
-  const segments = location.pathname?.split('/') ?? []; // ["", "alerting", "{sourceName}", "{name}]
+  const segments = location.pathname?.replace(subUrl, '').split('/') ?? []; // ["", "alerting", "{sourceName}", "{name}]
```

Comment:
There's some weird stuff going on here – but the gist is that this doesn't work with subpaths since it assumes a certain path structure.